rosdep/base.yaml

```
@@ -8339,6 +8339,7 @@ xtensor:
   ubuntu:
     bionic: [xtensor-dev]
     jammy: [libxtensor-dev]
+  nixos: [xtensor]
```

```suggestion
  nixos: [xtensor]
  ubuntu:
    bionic: [xtensor-dev]
    jammy: [libxtensor-dev]
```

Alphabetical ordering